### DISCUSSION TITLE
Is it possible to skip certain repositories?

### DISCUSSION THREAD

**Discussion by shenke-adsk** [mend/renovate-ce-ee#577] [Q&A] [answered]
As on-prem Renovate admins, we occasionally come across "bad repositories", which are repositories that Renovate does not run well on. For example, a repository with lots of NPM dependencies to resolve. Such repositories could bring the whole system down or create lots of noise in logs. Is there a way for admins to control what repos can be skipped?

**Comment by PhilipAbed** [comment]
have you tried `MEND_RNV_AUTODISCOVER_FILTER`? [docs](https://github.com/mend/renovate-ce-ee/blob/main/docs/configuration-options.md)

**Reply by PhilipAbed** [reply, marked as answer]
from docs 

>You can negate the regex by putting an ! in front. Only use a single negation and don't mix with other filters because all filters are combined with or. If using negations, all repositories except those who match the regex are added to the result:

```
{
  "autodiscoverFilter": ["!/project/.*/"]
}
```
